apps/financial-aid/web-osk/src/components/Estimation/Estimation.tsx

```diff
@@ -31,7 +31,7 @@ const Estimation = ({
   usePersonalTaxCredit,
 }: Props) => {
   const currentYear = format(new Date(), 'yyyy')
-  const currentMonth = parseInt(format(new Date(), 'MM'))
+  const nextMonth = months[new Date().getMonth() + 1].toLowerCase()
```

Comment:
I am guessing we are going to do this in a couple of places should we move `months[new Date().getMonth() + 1].toLowerCase()`to `utils` ?

Reply:
We are also getting current year in couple of places we should also move that to utils, it will be nice if we change the format 😄